### DISCUSSION TITLE
sending Embed to this see only you?

### DISCUSSION THREAD

**Discussion by whyrmph** [Rapptz/discord.py#7882] [Q&A] [answered]
Hello, such a question.
Is it possible to send an Embed so that only the user who clicked the button would see it? As an example, I add an image, which I mean by a message for the user only.
Thank you in advance for the answer.
![dcYpC](https://user-images.githubusercontent.com/57942975/163238131-cf5835ab-49fa-4d2f-ba1a-e1aa0644280c.png)

**Comment by LostLuma** [comment, marked as answer]
This is possible, it's called an ephemeral message.

You can set the `ephemeral` keyword argument on [`InteractionResponse.send_message`](https://discordpy.readthedocs.io/en/latest/interactions/api.html?highlight=interaction#discord.InteractionResponse.send_message) when sending your embed to send one, it's compatible with all the other options.

**Comment by DenverCoder1** [comment]
Use `await interaction.response.send_message` with `ephemeral=True` as LostLuma said.

Note, you can only do this when responding to a slash command, context menu command, modal submit, or component interaction (buttons/select menus)

**Comment by whyrmph** [comment]
OMG!!!!, it works, thank you so much!!!
